tests/integration_test/callback_service_test.py

```
@@ -1,14 +1,23 @@
 
 import pytest
 from unittest.mock import patch, AsyncMock
+
+from httpx import QueryParams
 from mongoengine import connect
+from blacksheep.contents import JSONContent
+from requests import Request
 
 from kairon import Utility
 from fastapi.testclient import TestClient
+from blacksheep.testing import TestClient
```

Ruff (0.8.2)

Redefinition of unused `TestClient` from line 12

(F811)